src/mumble/GlobalShortcut_win.cpp

```diff
@@ -20,10 +20,10 @@
 
 // 3rdparty/xinputcheck-src.
 #include <xinputcheck.h>
+// Tool snapshotting
+#include <tlhelp32.h>
 
 // We define a global macro called 'g'. This can lead to issues when included code uses 'g' as a type or parameter name (like protobuf 3.7 does). As such, for now, we have to make this our last include.
-#include "Global.h"
```

Comment:
As I mentioned, this should be restored, it's required to access global variables.